components/bl-list-component/src/list.js

```diff
@@ -0,0 +1,23 @@
+export function List({ options, items, type }) {
+  const ComponentType = type || options.type;
+
+  return (
+    <ComponentType className="list">
+      {items?.map((item, i) => (
+        <Item listItem={item} key={i} options={options} />
```

Comment:
We don't need to pass full item into Item component using item object.
Create different props for content, children and so on.

Also, at onClick handler pass different context block for content and children